### DISCUSSION TITLE
Feature request: Youtube last frame grid removal

### DISCUSSION THREAD

**Discussion by adamhruby** [jordwest/news-feed-eradicator#434] [Ideas]
Getting rid of the related column is great, but we are still seduced to the limbic hijack clickhole by the last screen of each video. (See pic.)
What if the plugin can hit pause on the player (like 2s) right before the grid appears? Or put an overlay over the video (with quotes)?  

![image](https://user-images.githubusercontent.com/88742134/128925636-447e295b-d41d-44b2-bf29-0cf8b4c18edd.png)

**Comment by brandon-mork** [comment]
Great idea! In the meantime, [Unhook - Remove YouTube Recommended Videos ](https://chrome.google.com/webstore/detail/unhook-remove-youtube-rec/khncfooichmfjbepaaaebmommgaepoid) is an excellent and highly configurable extension focused on configuring YouTube. 

Edit: I also submitted a pull request implementing this feature though, for when the maintainer has time to review it and release the next version of the extension! 😄
